greenpithumb/humidity_sensor.py

```diff
@@ -0,0 +1,27 @@
+import dht11_exceptions
+
+class HumiditySensor(object):
+	"""Wrapper for humidity sensor."""
+
+	def __init__(self, dht11_result):
+		"""Creates a new HumiditySensor wrapper.
+
+		Args:
+			dht11_result: Result of a reading from a DHT11 humidity and
+			temperature sensor.   
+		"""
+		self._dht11_result = dht11_result
+
+	def get_humidity_level(self):
+		"""Returns humidity level."""
+
+		if self._dht11_result.error_code() == 1:
+			raise dht11_exceptions.MissingDataError()
```

Comment:
For this and for `IncorrectCRCError` we should include human-readable messages like

```python
raise MissingDataError('DHT11 sensor reported missing data')
```